Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -447,10 +447,17 @@ public struct DocumentationNode {
                     for: SymbolGraph.Symbol.Location.self
                 )?.url()
 
-                for comment in docCommentDirectives {
-                    let range = docCommentMarkup.child(at: comment.indexInParent)?.range
+                for directive in docCommentDirectives {
+                    let range = docCommentMarkup.child(at: directive.indexInParent)?.range
                     
-                    guard BlockDirective.allKnownDirectiveNames.contains(comment.name) else {
+                    // Don't throw a warning if a directive is not included in allKnownDirectiveNames
```

Comment:
Just some additional context that might be helpful here...

```suggestion
                    // Only throw warnings for known directive names.
                    //
                    // This is important so that we avoid throwing warnings when building
                    // Objective-C/C documentation that includes doxygen commands.
```